### ISSUE TITLE
Graph not displaying 

### ISSUE THREAD

**Issue by WalCan** [nightscout/nightguard#180]
The graph is not showing in the main page of the app. I’m using the latest version of the app, 3.6.5.761, and the latest version of Nightscout, 14.2.5. Any assistance would be greatly appreciated.

**Comment by dhermanns** [comment]
Hi -hard to say. You can leave me your URI:

issue180.hermanns@spamgourmet.com

**Comment by WalCan** [comment]
Thanks for getting back to me. I’d prefer not to share the URL if possible,
however, I understand that makes it more difficult to troubleshoot. Are
there any settings I can check? I’m using access tokens and have given
Nightguard the “careportal” role. It is pulling in the BG value fine, just
not the graph.

Thanks.

**Comment by dhermanns** [comment]
Sounds ok. I would expect this is due to your security settings. But thats
just a wild guess. You could try to enable tokenless read access and see if
that helps.

WalCan ***@***.***> schrieb am Mi. 2. Feb. 2022 um 17:54:

> Thanks for getting back to me. I’d prefer not to share the URL if possible,
> however, I understand that makes it more difficult to troubleshoot. Are
> there any settings I can check? I’m using access tokens and have given
> Nightguard the “careportal” role. It is pulling in the BG value fine, just
> not the graph.
>
> Thanks.
>

**Comment by WalCan** [comment]
Hi,

You were absolutely correct, I enabled read tokenless access to test and it
immediately started working. Is it possible to have the app working with
token based access or is it currently limited to tokenless?

Thanks again for your help.

**Comment by dhermanns** [comment]
As far as I know, other people are using token based access. So it should
basically work.

WalCan ***@***.***> schrieb am Do. 3. Feb. 2022 um 04:50:

> Hi,
>
> You were absolutely correct, I enabled read tokenless access to test and it
> immediately started working. Is it possible to have the app working with
> token based access or is it currently limited to tokenless?
>
> Thanks again for your help.
>

**Comment by WalCan** [comment]
Ok, thanks. Any other suggestions on what I may need to tweak to get it
working?

Thanks.

**Comment by dhermanns** [comment]
No - not really.

WalCan ***@***.***> schrieb am Do. 3. Feb. 2022 um 07:28:

> Ok, thanks. Any other suggestions on what I may need to tweak to get it
> working?
>
> Thanks.
>

**Comment by dhermanns** [comment]
Let us know if you have any more input/feedback for us...

**Comment by tast3rmonial** [comment]
Dear ,

We are excited to give you an update on Tastermonial.

Today we have initiated a way to evaluate food claims and for customers to get a prescription for continuous glucose monitor (CGM) sensors and experiment with different foods using our app. 

We are actively working to improve your user experiences in:

👉 Ordering a CGM

We realized the uncertainty of CGM prices in major pharmacies around the US. Therefore, our team is now working on a solution to order CGM online and ship directly to you at a competitive price.

👉 Learning from your CGM food experiment data

[See video of current app's abiity](https://youtu.be/xtaoSxdqq6U)

Here's a peek into upcoming Tastermonial app's upgrade in the next few weeks:

- the ability to run food pairing experiments wtih CGM
- automatic syncing of CGM data to view and share the blood sugar experiments
- You'll be able to use our app to scan and evaluate more foods in your pantry (we're expanding our food database)
 We'd be happy to gather your feedback that could help us in prioritizing the next launches. Click the button below to let us know your needs.

[Customer ***@***.***&firstname=&lastname=)

 

Thanks for reading!

Bude & Tastermonial Team

[Facebook](https://www.facebook.com/tastermonial)
[Instagram](https://www.instagram.com/tastermonial)

No longer want to receive these emails? [Unsubscribe](https://manage.kmail-lists.com/subscriptions/unsubscribe?a=TvyBXH&c=01FTWCYKVXWVNSNSJPKK228DB2&k=6d25c58c19d58e114a7bb78739a29b9b&m=WUxwcT&r=RaxVQvE).
TASTERMONIAL
